### PR TITLE
My next huge PR

### PR DESCRIPTION
**Pull request by jarednorman**
What does it do?
----------------

Adds new frontend interface

How does it work?
-----------------

Anything to consider when deploying?
------------------------------------

Absolutely not.

### CODE REVIEW

**Review by stewart — DISMISSED** [review]
Good work! I completely approve of these changes!

**Review by stewart — APPROVED** [review]
I really wish you'd make this good before PR-ing it.

**Comment by jarednorman** [comment]
@stewart I really wish u were not a Hanzo main

**Comment by jarednorman** [comment]
@forkata merge it

File: app/views/home/index.html.haml [hunk 1]
```diff
@@ -1,12 +1 @@
...
-      %h2 Leaderboard
-      = render "shared/leaderboard", players: ranked_players
-
-    .homepage-chart.porkbox-light
-      %canvas.elo-chart{ data: { labels: elo_range.map{ |x| l(x, format: :short) }.to_json, ratings: elo_data.to_json } }
-
-    .homepage-recent-matches.porkbox-light
-      %h2 Recent matches
-      = render "matches/list", matches: recent_matches
+%h1 Get ☑️ rekt Chris
```
Comment (forkata) [inline app/views/home/index.html.haml]: :metal: :ping_pong: :ok_hand:

**Comment by forkata** [comment]
I also approve of all the work here! Just one question, will the dashboard still work?

**Comment by jarednorman** [comment]
It will.

**Comment by jarednorman** [comment]
But don't merge this.

**Pull request closed without merging**